File: Exercise/Medium/02. Sum of List/Code/Sum of List.cpp
#include <iostream>
#include <string>
#include <sstream>

using namespace std;

void sumList() {
    string line;
    getline(cin, line);
    istringstream istr(line);
    
    int num;
    int sum;
    while (istr >> num) {
        sum += num;
    }

    cout << sum << endl;
}

int main() {
    sumList();

    return 0;
}

File: Exercise/Medium/02. Sum of List/Code/Sum of List.py
import sys

def sum_list():
    line = sys.stdin.readline().strip()
    istr = line.split()
    
    total_sum = 0
    for num in istr:
        total_sum += int(num)

    print(total_sum)


if __name__ == "__main__":
    sum_list()